app/Task.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name', 'priority', 'is_completed', 'user_id',
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo('App\User');
```

Comment:
Umesto da se ovako prosledjuje putanja, mozes importovati User model i pozvati `return $this->belongsTo(User::class);`. Ukoliko se putanja nekada izmeni, ovaj metod nece morati da se menja. Izmeniti i u ostatku koda (ukoliko se pojavljuje jos negde).